Tests/LibWeb/Text/input/UIEvents/special-keys.html

```diff
@@ -0,0 +1,17 @@
+<input id="input" />
+<script src="../include.js"></script>
+<script>
+    const SPECIAL_KEYS = ["Escape"];
+
+    test(() => {
+        let input = document.getElementById("input");
+
+        input.addEventListener("keydown", e => {
+            println(`${e.key}`);
+        });
+
+        SPECIAL_KEYS.forEach(key => {
+            internals.sendKey(input, "Escape");
```

Comment:
(did you mean to use `key` here?)

Reply:
Oh woops, yes that should have been `key`. This was moved to `Tests/LibWeb/Text/input/UIEvents/KeyEvent-functional-keys.html` in commit 4fcaeabe1a6acd9f4d5fd811a36f2e86748b2a72, where it sends the right key.